libqfieldsync/offliners.py

```diff
@@ -351,6 +362,7 @@ def _convert_to_offline_project(
         offline_gpkg_path: str,
         offline_layers: Optional[List[QgsMapLayer]],
         bbox: Optional[QgsRectangle],
+        exported_project_title: Optional[str] = "",
```

Comment:
```suggestion
        exported_project_title: str = "",
```

This is not optional anymore.